test/test.js

```diff
@@ -875,6 +877,180 @@ test('transport.consume() with a non object appData rejects with TypeError', asy
 		.toThrow(TypeError);
 }, 500);
 
+test('transport.consume() with mutiple video and single audio(fix mid exception)', () =>
+{
```

Comment:
It would be great having some meaningful `expect`  in this test.

Reply:
I know that not throwing is an assert in this case, but adding few `expect`s would be more future proof.

Reply:
I have added some exception tests